### ISSUE TITLE
"Post As" feature

### ISSUE THREAD

**Issue by gsf00001** [SocialEngine/phpv4-issues#558]
This may be more of a 3rd-Party request (ex. SEAO has Groups, Directory/Pages, Directory/Businesses), but even for basic SE out of the box, this would be helpful.  When a User Admins a Group (or multiple groups) it would be great if they could choose 'who they are' when posting, and this would be reflected throughout.

**Comment by Elshara** [comment]
I think I understand what you are saying. You want to be able to post updates as a group name, so that they can be distinguished from the updates people post in the group.
Facebook and PHPFox have this functionality by default. It's something I am highly against, and don't understand the purpose of. But since its infamous popularity is such that people like it, I can kind of see a use for it. Or, if you want to be more involved, how about being able to post something as a specific member level or category depending on whatever the setting is site wide.
In other words, being able to post as the role of the website group or site wide position you carry, not just as you, but as a title you inherit as a part of the group or website itself. Nice idea, I just don't really think it's implemented all that well yet.
Social Engine Add On makes a lot of features like this possible as extensions. As you say, an interesting core suggestion. Make the group of users, its own contributor on the website as a prolific one. Good advertising move.

**Comment by Khushboo016** [comment]
Hi @gsf00001, Currently, there are no options available in core SE PHP to be able to log-in as a page or as a group, etc, so the feature to be able to choose while posting that whether you are a page or group, etc. have no significance. So, this feature is more useful in case of third party plugins you're referring here and if those 3rd plugins are missing this feature, you can suggest the same feature to their respective developers.

Thanks & Regards!

**Comment by Elshara** [comment]
It isn't something I'd use anyways.

**Comment by gsf00001** [comment]
@Elshara 
-->It's something I am highly against, and don't understand the purpose of.
I understand where you're coming from, but here's an example: Some sites are 'friends' based (so you either recognize the name or not) others are focused more on Groups (or with some 3rd-Party Plugins, Groups, Pages, Businesses, etc.).  In that case, it's often useful to know what 'entity' posted the content, since knowing that 'John Smith' posted something may be meaningless to some people, but knowing that it was posted by 'Pepsi Co' may have more meaning/usefulness.

@Khushboo016 
I have contacted SEAO and the reply is that since it's not supported internally by SE that they won't touch this.  It seems they want something system-wide to build upon (especially since SE Groups could use this - not just 3rd-Party Devs/Plugins).

**Comment by Elshara** [comment]
Yeah and I find it kind of concerning that if a third party won't
touch this basic feature, let alone give you a similar response to the
actual social engine development team, when you ask both for help on
this. It kind of makes you wonder who is the real developers of social
engine? You don't see them on here. I think pressuring them about it
might yield results eventually. Maybe if we got other people
interested in the feature, they might touch it. Nothing is impossible.
PHPfox has this feature built in and the lead developer, founder and
owner of PHPfox now works for social engine. And he built the very
same feature for PHPfox.

**Comment by DonnaScriptTechs** [comment]
This issue was moved to SocialEngine/phpv4-feature-requests#38

**Comment by DonnaScriptTechs** [comment]
In our efforts to streamline our processes for bug fixes and feature implementation, we are separating our issue tracker and feature request section. This has been moved to our feature area where you are subscribed automatically in order to get updates. Thank you for your contribution.
